docs/api/transaction/get-mempool-transactions.example.json

```diff
@@ -0,0 +1,391 @@
+      "token_transfer": {
+        "recipient_address": "STXW04Y2MCA7SHXTV32AD332JDT3JRB2YNC2Y0M1",
+        "amount": "500000",
+        "memo": "0x46617563657400000000000000000000000000000000000000000000000000000000"
+      }
+    },
+    {
+      "tx_id": "0xb31df5a363dad31723324cb5e0eefa04d491519fd30827a521cbc830114aa50c",
+      "tx_status": "pending",
+      "tx_type": "token_transfer",
+      "receipt_time": 1598288370,
+      "receipt_time_iso": "2020-08-24T16:59:30.000Z",
+      "fee_rate": "180",
+      "sender_address": "STB44HYPYAT2BB2QE513NSP81HTMYWBJP02HPGK6",
+      "sponsored": false,
+      "post_condition_mode": "deny",
+      "token_transfer": {
+        "recipient_address": "ST1GY25DM8RZV4X15X07THRZ2C5NMWPGQWKFGV87F",
+        "amount": "500000",
+        "memo": "0x46617563657400000000000000000000000000000000000000000000000000000000"
+      }
+    },
+    {
+      "tx_id": "0x5bed8e3f801cb4e2c74d2815a092f7c1c6a35f2fce4a80c80ca70848d34cb395",
+      "tx_status": "pending",
+      "tx_type": "token_transfer",
+      "receipt_time": 1598282326,
+      "receipt_time_iso": "2020-08-24T15:18:46.000Z",
+      "fee_rate": "180",
+      "sender_address": "ST1PS1KF93VBY5A1JV7TM66KN046KP3E3761DBSAG",
+      "sponsored": false,
+      "post_condition_mode": "deny",
+      "token_transfer": {
+        "recipient_address": "ST2H33S8SEY27QKEKQKR6S5PECYPKY45CQYGGQR8X",
+        "amount": "1000",
+        "memo": "0x00000000000000000000000000000000000000000000000000000000000000000000"
+      }
+    },
+    {
+      "tx_id": "0x9916036fde08a207e581cdcabc18ff55469861cb81194ab0e3e7c9a02cd5a17c",
+      "tx_status": "pending",
+      "tx_type": "smart_contract",
+      "receipt_time": 1598258958,
+      "receipt_time_iso": "2020-08-24T08:49:18.000Z",
+      "fee_rate": "2000",
+      "sender_address": "ST2R1XSFXYHCSFE426HP45TTD8ZWV9XHX2SRP3XA8",
+      "sponsored": false,
+      "post_condition_mode": "deny",
+      "post_conditions": [],
+      "smart_contract": {
+        "contract_id": "ST2R1XSFXYHCSFE426HP45TTD8ZWV9XHX2SRP3XA8.test-loans-1",
+        "source_code": "(define-data-var stx-loaned int 0)\n(define-data-var lockup-period int 0)\n(define-data-var stx-return int 0)\n(define-public (get-stx-return (stx uint) (months uint))\n    (ok\n        (begin\n            (var-set stx-loaned (to-int(stx))\n            (var-set lockup-period (to-int(months))\n            (calculate-stx-return)\n            (print (var-get stx-return))\n            (transfer-to-server)\n        )\n    )\n)\n(define-private (calculate-stx-return)\n    (ok\n        (begin\n            (var-set stx-return (- (+ (* (/ (* (var-get stx-loaned) 5) 100) (var-get lockup-period)) (var-get stx-loaned)) 5))\n        )\n    )\n)\n(define-private (transfer-to-server)\n  (begin\n    (unwrap-panic (stx-transfer? (to-uint (var-get stx-loaned)) tx-sender 'ST2R1XSFXYHCSFE426HP45TTD8ZWV9XHX2SRP3XA8))\n    (ok (to-uint (var-get stx-loaned)))\n  )\n)"
```

Comment:
how do we feel about the length of these fields? This would be shown in the docs and will be hard to read anyways

Reply:
It doesn't show up in outputs docs, so think it's worth having a complete contract

Reply:
Wouldn't it show up on the right side of these docs as a sample? https://blockstack.github.io/stacks-blockchain-api/

Reply:
Only if you expand it 
![image](https://user-images.githubusercontent.com/1618764/91320652-17f5ab00-e7be-11ea-869a-4fe99d5ca5f1.png)

Reply:
yeah - and if you do, it's not formatted well enough to be readable. the benefit of this lengthy example would be outside of the docs then I'd assume. fine with me :)